scripts/ansible-runner

```diff
@@ -94,12 +94,18 @@ if [ -n "$SKIP_TAGS" ]; then
   EXTRA_ANSIBLE_ARGS="${EXTRA_ANSIBLE_ARGS} --skip-tags $(echo $SKIP_TAGS | jq -r '. | join(",")')"
 fi
 if [ "$AWS_INVENTORY" == "auto" ] && [ -n "$AWS_ACCESS_KEY_ID" ] || [ "${AWS_INVENTORY,,}" == "true" ]; then
+  # Render ec2.ini template from envvars
+  envsubst < /etc/ansible/hosts-template/ec2.ini.template > /etc/ansible/hosts/ec2.ini
+  mv /etc/ansible/hosts-template/ec2.py /etc/ansible/hosts/
   EXTRA_ANSIBLE_ARGS="${EXTRA_ANSIBLE_ARGS} -i /etc/ansible/hosts/ec2.py"
 fi
 if [ "$AZURE_INVENTORY" == "auto" ] && [ -n "$AZURE_SUBSCRIPTION_ID" ] || [ "${AZURE_INVENTORY,,}" == "true" ]; then
   if (( $(echo "${ANSIBLE_VERSION%.*} >= 2.7" |bc -l) )); then
+    # Render default.azure_rm.yml template from envvars
+    envsubst < /etc/ansible/hosts-template/default.azure_rm.yml.template > /etc/ansible/hosts/default.azure_rm.yml
     EXTRA_ANSIBLE_ARGS="${EXTRA_ANSIBLE_ARGS} -i /etc/ansible/hosts/default.azure_rm.yml"
   else
+    mv /etc/ansible/hosts-template/azure_rm.py /etc/ansible/hosts/
```

Comment:
@sdurrheimer can you check this part ?

Do we need this yml only when ansible > 2.7
and only the py when ansible < 2.7 ?

Reply:
Yes, the `YAML` is the inventory plugin (>2.7), the `.py` the inventory script that should work on every versions but lack support of Azure VirtualMachineScaleSet.